.taskcluster.yml

```diff
@@ -58,6 +58,39 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("grcov_test_task")}
```

Comment:
We should integrate code coverage directly in the normal test tasks, this way we don't run the same tests twice.

Reply:
grcov requires the `nightly` version of Rust to be used, while the normal test task could be used to run `rust-code-analyisis` on stable, in this way we test both the versions at the same time

Reply:
Sounds good, so let's use stable in the normal test task